pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -155,3 +156,31 @@ func removeAllLogs() error {
 func removeCRCHostEntriesFromKnownHosts() error {
 	return ssh.RemoveCRCHostEntriesFromKnownHosts()
 }
+
+// remove the following checks during release of v2.35.0 or after
```

Comment:
In my opinion, it's the kind of checks we have to keep forever, as we often have people upgrading from very old versions. 
Alternatively, in a few releases, we could switch to a preflight which would only check for the file existence, and advise the user to use `crc cleanup` if such a file exists.